asmcomp/asmgen.ml

```diff
@@ -212,7 +215,7 @@ let flambda_gen_implementation ?toplevel ~source_provenance ~backend ppf
           definition })
       (Symbol.Map.bindings constants)
   in
-  end_gen_implementation ?toplevel ~source_provenance ppf
+  end_gen_implementation ?toplevel ~source_provenance ppf None
```

Comment:
does this mean that `flambda`-compiled modules do not have a root map? Can this limitation be lifted?